afc-submit-wizard.js

```diff
@@ -59,6 +59,12 @@ var messages = {
 	"orestopic-placeholder": "Start typing to search for topics ...",
 	"orestopic-label": "Topic classifiers",
 	"orestopic-helptip": "Pick the topic areas that are relevant",
+	"bestsources-placeholder1": "Enter your first source here",
```

Comment:
Should be more clear on what to enter: we need either the url or a brief citation for offline sources (without using citation templates).